### PR TITLE
Feature/issues/286

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

File: Extension/lib/content-script/assistant/js/adguard-rules-constructor.js [hunk 1]
```diff
@@ -244,6 +255,34 @@ var AdguardRulesConstructorLib = (function (api) {
     };
 
     /**
+     * Constructs css selector for specified rule
+     *
+     * @param ruleText rule text
+     * @returns {string} css style selector
+     */
+    api.constructRuleCssSelector = function (ruleText) {
```
Comment (ameshkov) [inline Extension/lib/content-script/assistant/js/adguard-rules-constructor.js]: Can't we use CssFilterRule class here?

File: Extension/lib/content-script/assistant/js/adguard-rules-constructor.js [hunk 2]
```diff
@@ -244,6 +255,34 @@ var AdguardRulesConstructorLib = (function (api) {
...
+
+        if (index >= 0) {
+            return ruleText.substring(index + CSS_RULE_MARK.length, optionsIndex >= 0 ? optionsIndex : ruleText.length);
+        }
+
+        var s = ruleText.substring(0, optionsIndex);
+        s = s.replace(/[\|]|[\^]/g, '');
+
+        if (isValidUrl(s)) {
+            return '[src="' + s + '"]';
```
Comment (ameshkov) [inline Extension/lib/content-script/assistant/js/adguard-rules-constructor.js]: I suppose you should use this type of selector instead:
```
[src^=
```

File: Extension/lib/content-script/assistant/js/adguard-rules-constructor.js (hunk 1 above)
Comment (Mizzick) [inline Extension/lib/content-script/assistant/js/adguard-rules-constructor.js]: Nope, if will bring a lot of dependecies etc

File: Extension/lib/content-script/assistant/js/adguard-rules-constructor.js (hunk 2 above)
Comment (Mizzick) [inline Extension/lib/content-script/assistant/js/adguard-rules-constructor.js]: Okay, fixed

**Pull request merged**